### PR TITLE
Separate the samples and assets and update the README

### PR DESCRIPTION
**Pull request by splovyt**
I have updated the README with the VCPI urls and the tutorial button. In addition, I have separated the samples and assets.

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Test Examples (samples/test_examples/)
+
+This directory contains to subdirectories:
+* low_resolution
+* original
+
+The low resolution images can be used as input for the model, and the resulting high resolution images can be compared with the 'original' images to measure performance.
+
+The source of the images is the following:
```
Comment (xuhdev) [inline samples/README.md]: NIT: sources of the images are

File: samples/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Sample Details
+
+## Test Examples (samples/test_examples/)
```
Comment (xuhdev) [inline samples/README.md]: nit: `samples/test_examples/`

File: assets/README.md [hunk 3]
```diff
@@ -4,34 +4,4 @@
...
-- `astronaut.png`
-- `elephant.png`
-- `face.png`
-- `face_paint.png`
-- `food.png`
-- `palm_tree.png`
-- `penguin.png`
-- `woman.png`
\ No newline at end of file
+_Note: the finetuned model files are hosted on [IBM Cloud Object Storage](http://max-assets.s3.us.cloud-object-storage.appdomain.cloud/max-image-resolution-enhancer/1.0/assets.tar.gz)._
```
Comment (xuhdev) [inline assets/README.md]: nit: fine-tuned, maybe?

**Comment by MLnick** [comment]
> A general question: For testing purpose, is it also useful to have some image in other formats, such as jpeg, tiff, along the side?

Good point - we have done this for models where we do support various input formats (see e.g.  https://github.com/IBM/MAX-ResNet-50/blob/master/tests/test.py#L45)

File: samples/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Sample Details
+
+## Test Examples (`samples/test_examples/`)
```
Comment (MLnick) [inline samples/README.md]: I would actually do this:

```md
## Test Examples

The `samples/test_examples/` folder contains two subfolders:
```

**Comment by splovyt** [comment]
I have added a test case for TIFF and JPEG encoded images.

File: tests/test.py [hunk 5]
```diff
@@ -65,25 +65,43 @@ def test_predict():
...
+    im = call_model(file_path='samples/test_examples/low_resolution/elephant.png')
     assert im.size == (868, 1392)
 
 
+def test_predict_other_formats():
+    '''Check the prediction output of JPEG and TIFF converted images.'''
+
+    # save img as jpeg
+    Image.open('samples/test_examples/low_resolution/woman.png').save(
+        'samples/test_examples/low_resolution/woman.jpeg', 'JPEG')
```
Comment (xuhdev) [inline tests/test.py]: How about something like this:
```python
with Image.open('samples/test_examples/low_resolution/woman.png') as img:
    for ext in ('jpeg', 'tiff'):
        with TemporaryFile(suffix='.' + ext) as tmp_img:
            img.save(new_img)
            im = call_mode(file_path=tmp_img.name)
            assert im.size == (426, 636)
```

Comment (splovyt) [reply tests/test.py]: Thanks Hong!!

File: tests/test.py [hunk 6]
```diff
@@ -88,19 +89,12 @@ def test_predict():
...
-
-    im = call_model(file_path='samples/test_examples/low_resolution/woman.jpeg')
-    assert im.size == (424, 636)
-
-    im = call_model(file_path='samples/test_examples/low_resolution/woman.tiff')
-    assert im.size == (424, 636)
+    with Image.open('samples/test_examples/low_resolution/woman.png') as img:
+        for ext in ('jpeg', 'tiff'):
+            with NamedTemporaryFile(suffix='.' + ext) as tmp_img:
+                img.save(tmp_img)
```
Comment (xuhdev) [inline tests/test.py]: I forgot to add the format: `img.save(tmp_img, ext)`. Using this occasion, we can perhaps also add `gif`.

**Pull request merged**